.github/ISSUE_TEMPLATE/bug_report_data.md

```diff
@@ -0,0 +1,96 @@
+---
+name: 🐛 Bug Report - 💾 Data
```

Comment:
Explain this please, because it doesn' make amy sense to me

Reply:
This is from what I lately learned at `vmangos` - the main type or issue or PR is about incorrect or missing data which is not aligned with the sniffs or what ever. It might be a bit different with WOTLK since there is only old sniff data - I just assume that the main Issues you face are still broken quests, incorrect data or simply missing data and not bugs with the core or subsystems itself.

Reply:
I mean we dont have sniffs and personally my first thoyght was that anything can be missing data, for example if a crrsture is missing immunities, is it a missing data or a cresture bug. Well both.

So I dont know :shrug:

Reply:
As stated above you should find a few very general types of bugreports users can use to precategorize them and give maintainers a visual feedback - my experience tells me that data related stuff is a very common on